langserver/handler_shared.go

```diff
@@ -25,9 +26,9 @@ type HandlerShared struct {
 
 // FindPackageFunc matches the signature of loader.Config.FindPackage, except
 // also takes a context.Context.
-type FindPackageFunc func(ctx context.Context, bctx *build.Context, importPath, fromDir string, mode build.ImportMode) (*build.Package, error)
+type FindPackageFunc func(ctx context.Context, bctx *build.Context, conn jsonrpc2.JSONRPC2, importPath, fromDir string, mode build.ImportMode) (*build.Package, error)
```

Comment:
Can you update the doc string to mention conn. It would also be nice if you made it the 2nd param, such that everything after the 2nd param matched FindPackage.